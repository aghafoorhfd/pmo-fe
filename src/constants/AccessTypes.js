export const ACCESS_TYPES = {
  SUPER_ADMIN: 'SUPER_ADMIN',
  ADMIN: 'ADMIN',
  PROJECT_MANAGER: 'PROJECT_MANAGER',
  RESOURCE_MANAGER: 'RESOURCE_MANAGER',
  CONFLICT_MANAGER: 'CONFLICT_MANAGER',
  GENERAL_USER: 'GENERAL_USER'
};
