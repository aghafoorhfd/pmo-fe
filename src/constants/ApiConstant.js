import { env } from '../configs/EnvironmentConfig';

export const REACT_APP_BASE_URL = env.REACT_APP_URL;

export const USER_SERVICE_BASE_URL = env.USER_SERVICE;
export const USER_SERVICE = `${USER_SERVICE_BASE_URL}/user_service/api/users`;
export const USER_PROFILE = `${USER_SERVICE}/user-profile`;
export const REGISTER_USER = `${USER_SERVICE}/register`;
export const FORGOT_PASSWORD = `${USER_SERVICE}/forgot-password`;
export const VERIFY_EMAIL = `${USER_SERVICE}/verify-email`;
export const SET_CREDENTIALS = `${USER_SERVICE}/credentials`;
export const LOGIN = `${USER_SERVICE}/login`;

export const AUTH_SERVICE_BASE_URL = env.AUTH_SERVICE;
export const AUTH_SERVICE = `${AUTH_SERVICE_BASE_URL}/auth_service/api/users`;
export const RESET_CREDENTIALS = `${AUTH_SERVICE}/reset-password`;

export const GLOBAL_SERVICE_BASE_URL = env.GLOBAL_SERVICE;
export const GLOBAL_SERVICE = `${GLOBAL_SERVICE_BASE_URL}/global_service/api`;

export const PROJECT_SERVICE_BASE_URL = env.PROJECT_SERVICE;
export const PROJECT_SERVICE = `${PROJECT_SERVICE_BASE_URL}/project_service/api`;
export const ADD_PROJECT = `${PROJECT_SERVICE}/projects`;
export const INVITE_STAKEHOLDERS = `${PROJECT_SERVICE}/projects/invite-stakeholders`;
export const PROJECT_RESOURCES = `${PROJECT_SERVICE}/projects/resources-stats`;
export const PROJECT_CONFLICTS = `${PROJECT_SERVICE}/projects/conflicts-stats`;
export const COMPANY_METHODOLOGY = `${PROJECT_SERVICE}/company/methodology`;

export const PAYMENT_SERVICE_BASE_URL = env.PAYMENT_SERVICE;
export const PAYMENT_SERVICE = `${PAYMENT_SERVICE_BASE_URL}/paymentgateway_service/api`;

export const SUBSCRIPTION_SERVICE_BASE_URL = env.SUBSCRIPTION_SERVICE;
export const SUBSCRIPTION_SERVICE = `${SUBSCRIPTION_SERVICE_BASE_URL}/subscription_service/api`;

export const RESOURCE_SERVICE_BASE_URL = env.RESOURCE_SERVICE_API_ENDPOINT_URL;
export const RESOURCE_SERVICE = `${RESOURCE_SERVICE_BASE_URL}/resource_service/api`;
export const RESOURCE_TEAM = `${RESOURCE_SERVICE}/resource-teams`;
export const RESOURCE_REQUEST = `${RESOURCE_SERVICE}/project-manager/resources`;

export const PAYMENT_CONFIRMATION_URL = '/app/dashboards/payment-confirmation';
