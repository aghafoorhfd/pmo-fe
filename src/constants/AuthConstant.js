export const AUTH_TOKEN = 'auth_token';
export const LOGIN = '/auth/login';
export const SIGN_UP = '/auth/register';
