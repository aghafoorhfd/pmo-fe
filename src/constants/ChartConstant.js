export const COLOR_1 = '#3e82f7'; // blue
export const COLOR_2 = '#04d182'; // cyan
export const COLOR_3 = '#ff6b72'; // volcano
export const COLOR_4 = '#ffc107'; // gold
export const COLOR_5 = '#a461d8'; // purple
export const COLOR_6 = '#fa8c16'; // orange
export const COLOR_7 = '#17bcff'; // geekblue
export const COLOR_8 = '#A5A5A5'; // grey
export const COLOR_9 = '#FFEF5E'; // orange
export const COLOR_10 = '#509CF5'; // light-blue
export const COLOR_11 = '#D0D5DD'; // light Grey
export const COLOR_12 = '#039855'; // green
export const COLOR_13 = '#D92D20'; // red

export const COLOR_1_LIGHT = 'rgba(62, 130, 247, 0.15)';
export const COLOR_2_LIGHT = 'rgba(4, 209, 130, 0.1)';
export const COLOR_3_LIGHT = 'rgba(222, 68, 54, 0.1)';
export const COLOR_4_LIGHT = 'rgba(255, 193, 7, 0.1)';
export const COLOR_5_LIGHT = 'rgba(139, 75, 157, 0.1)';
export const COLOR_6_LIGHT = 'rgba(250, 140, 22, .1)';
export const COLOR_7_LIGHT = 'rgba(23, 188, 255, 0.15)';

export const COLORS = [
  COLOR_1,
  COLOR_2,
  COLOR_3,
  COLOR_4,
  COLOR_5,
  COLOR_6,
  COLOR_7,
  COLOR_8,
  COLOR_9,
  COLOR_10,
  COLOR_11,
  COLOR_12,
  COLOR_13
];

export const COLORS_LIGHT = [
  COLOR_1_LIGHT,
  COLOR_2_LIGHT,
  COLOR_3_LIGHT,
  COLOR_4_LIGHT,
  COLOR_5_LIGHT,
  COLOR_6_LIGHT,
  COLOR_7_LIGHT
];

export const COLOR_AXES = '#edf2f9';
export const COLOR_TEXT = '#455560';

export const apexLineChartDefaultOption = {
  chart: {
    zoom: {
      enabled: false
    },
    toolbar: {
      show: false
    }
  },
  colors: [...COLORS],
  dataLabels: {
    enabled: false
  },
  stroke: {
    width: 3,
    curve: 'smooth',
    lineCap: 'round'
  },
  legend: {
    position: 'top',
    horizontalAlign: 'right',
    offsetY: -15,
    itemMargin: {
      vertical: 20
    },
    tooltipHoverFormatter(val, opts) {
      return `${val} - ${opts.w.globals.series[opts.seriesIndex][opts.dataPointIndex]}`;
    }
  },
  xaxis: {
    categories: []
  },
  grid: {
    xaxis: {
      lines: {
        show: true
      }
    },
    yaxis: {
      lines: {
        show: false
      }
    }
  }
};

export const apexAreaChartDefaultOption = { ...apexLineChartDefaultOption };

export const apexBarChartDefaultOption = {
  chart: {
    zoom: {
      enabled: false
    },
    toolbar: {
      show: false
    }
  },
  plotOptions: {
    bar: {
      horizontal: false,
      columnWidth: '25px',
      startingShapre: 'rounded',
      endingShape: 'rounded'
    }
  },
  colors: [...COLORS],
  dataLabels: {
    enabled: false
  },
  stroke: {
    show: true,
    width: 6,
    curve: 'smooth',
    colors: ['transparent']
  },
  legend: {
    position: 'top',
    horizontalAlign: 'right',
    offsetY: -15,
    inverseOrder: true,
    itemMargin: {
      vertical: 20
    },
    tooltipHoverFormatter(val, opts) {
      return `${val} - ${opts.w.globals.series[opts.seriesIndex][opts.dataPointIndex]}`;
    }
  },
  xaxis: {
    categories: []
  },
  fill: {
    opacity: 1
  },
  tooltip: {
    y: {
      formatter: (val) => `${val}`
    }
  }
};

export const apexPieChartDefaultOption = {
  colors: [...COLORS],
  plotOptions: {
    pie: {
      donut: {
        size: '60%',
        labels: {
          show: true
        }
      }
    }
  },
  labels: [],
  dataLabels: {
    enabled: true,
    formatter: (val) => `${val.toFixed(0)}%`
  },
  legend: {
    show: true,
    onItemHover: {
      highlightDataSeries: false
    }
  },
  chart: {
    fontFamily: 'Inter',
    foreColor: 'rgb(69, 85, 96)'
  },
  stroke: {
    width: 0
  }
};

export const apexSparklineChartDefultOption = {
  chart: {
    type: 'line',
    sparkline: {
      enabled: true
    }
  },
  stroke: {
    width: 2,
    curve: 'smooth'
  },
  tooltip: {
    fixed: {
      enabled: false
    },
    x: {
      show: false
    },
    y: {
      title: {
        formatter() {
          return '';
        }
      }
    },
    marker: {
      show: false
    }
  }
};
