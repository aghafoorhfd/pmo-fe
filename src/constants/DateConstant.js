/* eslint-disable camelcase */
export const DATE_FORMAT_YYYYMMdd = 'YYYYMMdd';
export const DATE_FORMAT_YYYY_MM_dd = 'YYYY-MM-dd';
export const DATE_FORMAT_YYYY_MM_DD = 'YYYY-MM-DD';
export const DATE_FORMAT_YYYY_MM_dd_Combined = 'YYYYMMddHHmm';
export const DATE_FORMAT_YYYY_MM_dd_HH_mm = 'YYYY-MM-dd HH:mm';
export const DATE_FORMAT_DD_MM_YYYY = 'DD-MM-YYYY';
export const DATE_FORMAT_DD_MM_YYYY_WITH_DOT = 'DD.MM.YYYY';
export const DATE_FORMAT_mm_dd_YYYY_WITH_SLASH = 'MM/dd/YYYY';
export const DATE_FORMAT_m_d_YYYY_WITH_SLASH = 'M/d/YYYY';
export const DATE_FORMAT_DD_MM_YYYY_WITH_SLASH = 'DD/MM/YYYY';
export const DATE_FORMAT_dd_MMM = 'ddMMM';
export const DATE_FORMAT_dd_MM_yy = 'ddMMyy';
export const DATE_FORMAT_dd_MMM_YYYY = 'ddMMMYYYY';
export const DATE_FORMAT_DD_MM_YYYY_COMBINED = 'DDMMYYYY';
export const DATE_FORMAT_DD_MM_YYYY_HH_mm_ss = "YYYY-MM-dd'T'HH:mm:ss";
export const DATE_FORMAT_DD_MM_YYYY_HH_mm_ss_sz = "YYYY-MM-dd'T'HH:mm:ss.SSS";
export const DATE_FORMAT_DD_MM_YYYY_HH_mm = 'dd-MM-YYYY HH:mm';
export const DATE_FORMATE_DDD_dd_MMM_YYYY_H_MM_SS = 'ddd, DD MMM YYYY h:mm:ss A Z';
export const DATE_FORMAT_HH_mm = 'HH:mm';
export const DATE_FORMAT_YYYY_MM_DDTHH_mm_ssZ = "YYYY-MM-DD'T'HH:mm:ssZ";
export const DATE_FORMAT_MMM_DD_YYYY_WITH_COMMA = 'MMM DD, YYYY';
export const DATE_FORMAT_DD_MMM_YYYY_WITH_SPACE = 'DD MMM YYYY';
export const DATE_FORMAT_DD_MMM_WITH_SPACE = 'DD MMM';
export const DATE_FORMAT_MM_DD_WITH_SLASH = 'MM/DD';
export const DATE_FORMAT_dd_MMM_WITH_SPACE = 'DD MMM';
export const DATE_FORMAT_YYYY_MM_WITH_DASH = 'YYYY-MM';
export const DATE_FORMAT_D_MMM = 'D MMM';
export const DATE_FORMAT_D_MMM_YEAR = 'D MMM YYYY';
export const DATE_FORMAT_MM_DD_YYYY_HH_MM_SS = 'MM/DD/YYYY HH:mm:ss';
export const DATE_FORMAT_MM_DD_YYYY_WITH_SLASH = 'MM/DD/YYYY';
export const DATE_FORMAT_MM_DD_YYYY = 'MM-DD-YYYY';
export const DATE_FORMAT_MMM_D = 'MMM D';
export const DATE_FORMAT_MMM_D_YEAR = 'MMM D YYYY';
export const DATE_FORMAT_MM_YYYY = 'MM-YYYY';
export const DATE_FORMAT_MMM_DD_WITH_SPACE = 'MMM DD ';
export const DAY_TIME_DD_HH_MM = 'ddd h:mm A';
