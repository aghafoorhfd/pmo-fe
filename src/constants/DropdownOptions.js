import i18n from 'i18next';

const { t } = i18n;

export const USER_FORM_ACCESSTYPE = [
  {
    value: 'ADMIN',
    label: t('component.constant.dropDownOptions.admin')
  },
  {
    value: 'PROJECT_MANAGER',
    label: t('component.constant.dropDownOptions.projectManager')
  },
  {
    value: 'RESOURCE_MANAGER',
    label: t('component.constant.dropDownOptions.resourceManager')
  },
  {
    value: 'GENERAL_USER',
    label: t('component.constant.dropDownOptions.generalUser')
  }
  // Commenting the EXECUTIVE role for now It will be visible in future
  // {
  //   value: 'EXECUTIVE',
  //   label: t('component.constant.dropDownOptions.executive')
  // }
];

export const RESOURCE_CAPACITY_OPTIONS = [
  {
    value: 100,
    label: '100%'
  },
  {
    value: 50,
    label: '50%'
  }
];

export const TABLE_ROWS = ['10', '20', '50', '100'];

export const BILLING_CYCLE_OPTIONS = [
  { label: t('component.product.admin.monthly'), value: 'Monthly' },
  { label: t('component.product.admin.yearly'), value: 'Yearly' },
  { label: t('component.pricing.package.type.freemium'), value: 'Free' }
];

export const PLAN_TYPE_OPTIONS = [
  { label: t('component.analytics.dropDown.web'), value: 'WEB' },
  { label: t('component.analytics.dropDown.b2b'), value: 'B2B' }
];

export const STATUS_OPTIONS = [
  { label: t('component.analytics.dropDown.active'), value: 'ACTIVE' },
  { label: t('component.analytics.dropDown.inActive'), value: 'IN_ACTIVE' }
];

export const JIRA_WORK_TYPE = [
  { label: t('component.jira.dropDown.userStory'), value: 'USER_STORY' },
  { label: t('component.jira.dropDown.epic'), value: 'EPIC' },
  { label: t('component.jira.dropDown.initiative'), value: 'INITIATIVE' }

];
