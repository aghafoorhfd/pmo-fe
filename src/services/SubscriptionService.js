import fetch from 'auth/FetchInterceptor';
import {
  SUBSCRIPTION_SERVICE
} from 'constants/ApiConstant';

const SubscriptionService = {
  addPlan(data) {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/plans`,
      method: 'post',
      data
    });
  },

  getAllCustomersPlans(pageNumber, pageSize, uri = '') {
    const filterAndParam = uri && `?filterAnd=${uri}`;
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/plans/customers/${pageNumber}/${pageSize}${filterAndParam}`,
      method: 'get'
    });
  },

  getAllB2BPlans(pageNumber, pageSize) {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/plans/${pageNumber}/${pageSize}`,
      method: 'get'
    });
  },

  getPlans() {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/plans/web?isAdmin=false`,
      method: 'get'
    });
  },

  getActiveSubscription() {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/subscriptions/active`,
      method: 'get'
    });
  },

  getInvoice(params) {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/plans/invoice`,
      method: 'get',
      params
    });
  },

  getSubscriptionsBilling() {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/subscriptions/billing`,
      method: 'get'
    });
  },
  getPendingInvoice() {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/invoice`,
      method: 'get'
    });
  },
  upgradeEnterprisePlan(payload) {
    return fetch({
      url: `${SUBSCRIPTION_SERVICE}/plans/upgrade-enterprise-plan`,
      method: 'put',
      data: payload
    });
  }

};
export default SubscriptionService;
