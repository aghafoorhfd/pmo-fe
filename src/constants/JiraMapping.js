import { Tooltip, Typography } from 'antd';
import moment from 'moment';
import { getJiraBacklog, getJiraIssueMap, getJiraStories } from 'utils/jiraUtils';
import TooltipContent from 'views/app-views/dashboards/project-manager/jira-timeline/JiraTooltip';
import { t } from 'i18next';
import { setStagesDependencies } from 'utils/utils';
import { DATE_FORMAT_DD_MM_YYYY } from './DateConstant';

const { Paragraph } = Typography;

export const JiraMapping = {
  mapping: {
    start: (data) => moment(data.startDate, DATE_FORMAT_DD_MM_YYYY).startOf('day').toDate(),
    end: (data) => moment(data.endDate, DATE_FORMAT_DD_MM_YYYY).endOf('day').toDate(),
    id: ({ stageName, subStageName, projectName }) => stageName || (`${projectName}-${subStageName}`),
    theme: 'theme',
    stageName: ({ stageName }) => (
      <Tooltip className="gantt-chart-stage" title={stageName}>
        <Paragraph ellipsis>
          {stageName}
        </Paragraph>
      </Tooltip>
    ),
    subStageName: ({ subStageName }) => (
      <Tooltip className="gantt-chart-stage" title={subStageName}>
        <Paragraph className="cursor-pointer" ellipsis>
          {subStageName}
        </Paragraph>
      </Tooltip>
    ),
    hideChildren: true,
    team: 'team',
    type: ({ subStages }) => (subStages?.length ? 'project' : 'task'),
    dependencies: ({
      dependencies,
      subStagesDependencies,
      projectName
    }) => setStagesDependencies(dependencies, subStagesDependencies, projectName),
    jiraEpic: 'jiraEpic',
    barBackgroundColor: '#ffbb54',
    project: ({ projectName }) => projectName || null,
    styles: {
      backgroundColor: (data) => data.outlook
    },
    jiraIssues: 'jiraIssues',
    barText: (data) => {
      const { jiraIssues = {} } = data || {};
      const jiraStories = getJiraStories(Object.values(jiraIssues));
      return jiraStories?.length ? t('component.jira.chart.numberOfIssues', { count: jiraStories?.length }) : null;
    }
  },
  default: {
    stageName: '',
    subStageName: '',
    team: '',
    dependencies: [],
    jiraEpic: '',
    theme: '',
    jiraIssues: []
  }
};

export const jiraTimelineHeaders = (handleBacklogClick) => [
  { key: 'stageName', title: t('component.jira.chart.column.stageName') },
  { key: 'subStageName', title: t('component.jira.chart.column.subStageName') },
  { key: 'team', title: t('component.jira.chart.column.team') },
  { key: 'jiraEpic', title: t('component.jira.chart.column.jiraEpic') },
  {
    title: t('component.jira.chart.column.backlog'),
    render: (task) => {
      const { jiraIssues = {} } = task || {};
      const jiraIssuesMap = getJiraIssueMap(jiraIssues);
      const backlog = getJiraBacklog(jiraIssuesMap);
      return (
        <div
          className="jira-backlog-column">
          <Tooltip
            title={TooltipContent({ task, showBacklogOnly: true })}
            placement="right"
            overlayClassName="jira-overlay-tooltip"
            showArrow={false}>
            <span
              role="button"
              tabIndex={0}
              onClick={() => handleBacklogClick(task, true)}>
              {backlog.length ? t('component.jira.chart.numberOfIssues', { count: backlog?.length }) : null}

            </span>
          </Tooltip>
        </div>
      );
    }
  }
];
