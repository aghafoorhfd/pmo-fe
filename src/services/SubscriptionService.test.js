import fetch from 'auth/FetchInterceptor';
import { SUBSCRIPTION_SERVICE } from 'constants/ApiConstant';
import SubscriptionService from './SubscriptionService';

jest.mock('auth/FetchInterceptor');
jest.mock('utils/utils');

describe('Subscription Service', () => {
  afterEach(() => jest.clearAllMocks());
  it('should call get Web Plans ', () => {
    SubscriptionService.getPlans();

    expect(fetch).toBeCalledWith({
      method: 'get',
      url: `${SUBSCRIPTION_SERVICE}/plans/web`
    });

    expect(fetch).toHaveBeenCalledTimes(1);
    expect(fetch).toMatchSnapshot();
  });

  it('should call get All Customer Plans ', () => {
    SubscriptionService.getAllCustomersPlans(1, 1);

    expect(fetch).toBeCalledWith({
      method: 'get',
      url: `${SUBSCRIPTION_SERVICE}/plans/customers/1/1`
    });

    expect(fetch).toHaveBeenCalledTimes(1);
    expect(fetch).toMatchSnapshot();
  });

  it('should call get invoiceId and amount to be charged with.', () => {
    SubscriptionService.getInvoice({
      planPackageId: 'a474055c-8187-4df2-9be3-421acd02e96a',
      licensesToPurchase: 101
    });

    expect(fetch).toBeCalledWith({
      method: 'get',
      url: `${SUBSCRIPTION_SERVICE}/plans/web`
    });

    expect(fetch).toHaveBeenCalledTimes(1);
    expect(fetch).toMatchSnapshot();
  });

  it('should call get active subscription plan', () => {
    SubscriptionService.getActiveSubscription();

    expect(fetch).toBeCalledWith({
      method: 'get',
      url: `${SUBSCRIPTION_SERVICE}/subscriptions/active`
    });

    expect(fetch).toHaveBeenCalledTimes(1);
    expect(fetch).toMatchSnapshot();
  });

  it('should call get subscriptions billing', () => {
    SubscriptionService.getSubscriptionsBilling();

    expect(fetch).toBeCalledWith({
      method: 'get',
      url: `${SUBSCRIPTION_SERVICE}/subscriptions/billing`
    });

    expect(fetch).toHaveBeenCalledTimes(1);
    expect(fetch).toMatchSnapshot();
  });

  it('should call upgrade enterprise plan', () => {
    SubscriptionService.upgradeEnterprisePlan();

    expect(fetch).toBeCalledWith({
      method: 'put',
      url: `${SUBSCRIPTION_SERVICE}/plans/upgrade-enterprise-plan`
    });

    expect(fetch).toHaveBeenCalledTimes(1);
    expect(fetch).toMatchSnapshot();
  });
});
