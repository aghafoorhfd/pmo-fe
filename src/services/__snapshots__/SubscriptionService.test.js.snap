// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscription Service should call get All Customer Plans  1`] = `
[MockFunction] {
  "calls": [
    [
      {
        "method": "get",
        "url": "undefined/subscription_service/api/plans/customers/1/1",
      },
    ],
  ],
  "results": [
    {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;

exports[`Subscription Service should call get Web Plans  1`] = `
[MockFunction] {
  "calls": [
    [
      {
        "method": "get",
        "url": "undefined/subscription_service/api/plans/web",
      },
    ],
  ],
  "results": [
    {
      "type": "return",
      "value": undefined,
    },
  ],
}
`;
